modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/frontend/taglib/servlet/taglib/ActionsObjectDefinitionsScreenNavigationEntry.java

```diff
@@ -61,20 +53,14 @@ public String getCategoryKey() {
 
 	@Override
 	public String getEntryKey() {
-		return ObjectDefinitionsScreenNavigationEntryConstants.
-			ENTRY_KEY_ACTIONS;
+		return getCategoryKey();
```

Comment:
@RenanUmV Could you also extract this method to the base class, as all the sub classes of `BaseObjectDefinitionScreenNavigationEntry` have this method delegating to `getCategoryKey()`.